cpp/ql/lib/semmle/code/cpp/Concept.qll

```diff
@@ -6,9 +6,130 @@ import semmle.code.cpp.exprs.Expr
 
 /**
  * A C++ requires expression.
+ *
+ * For example, with `T` and `U` template parameters:
+ * ```cpp
+ * requires (T x, U y) { x + y; };
+ * ```
  */
 class RequiresExpr extends Expr, @requires_expr {
-  override string toString() { result = "requires ..." }
+  override string toString() { result = "requires(...) { ... }" }
```

Comment:
Nit: Consider something like `if exists(getAParameter()) then result = "requires(...) { ... }" else result = "requires { ... }" }`

Reply:
Yes, I thought about that. However, I cannot really differentiate between empty and missing parameter lists, so I wasn't really sure about that.

Reply:
Done.